mica/vv/tests/test_vv.py

```diff
@@ -30,3 +30,9 @@ def test_run_vv():
     obi = process.get_arch_vv(2121)
     assert np.allclose(obi.info()['sim']['max_d_dy'], 0.002197265625)
 
+def test_run_vv_omitted_slot():
+    obi = process.get_arch_vv(19991, version='last')
```

Comment:
These don't explicitly check anything, but I don't think that is a problem if really I'm just testing the routine didn't thrown an unhandled exception.